Jenkinsfile

```diff
@@ -1,2 +1,5 @@
-@Library(value="oicr", changelog=false) _
-arranger_api{}
\ No newline at end of file
+@Library(value="kids-first/aws-infra-jenkins-shared-libraries-develop", changelog=false) _
```

Comment:
Did you wanted to keep -develop shared library?

Reply:
Thanks, I missed that. It's fixed now